README.md

```diff
@@ -0,0 +1,26 @@
+## About
+
+This is a trainer for The Legend of Zelda: Ocarina of Time. It is compatible with version 1.0 and 1.2 of the NTSC release.
+
+## Dependencies
+* The n64-toolchain is required for building it can be found [here](https://github.com/glankk/n64).
+* In installing the n64-toolchain, Code::Blocks IDE should be configured to support targeting the n64. Alonside Code::Blocks, BizHawk should also be installed and configured, including all environment variables.
+
+## Building
```

Comment:
With the new changes making this project multi-target, the toolchain should be modified to better support these changes. Since that would soon make this information obsolete, I suggest delaying the addition of a readme until those details have been sorted out.